Tests/ApodiniTests/GRPCSemanticModelBuilderTests.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  GRPCSemanticModelBuilder.swift
```

Comment:
Nit: To be consistent, maybe renaming this test file to `GRPCInterfaceExporterTests.swift` (and the class accordingly)?